app/lib/pdf_filler/nj1040_pdf.rb

```diff
@@ -54,7 +54,19 @@ def hash_for_pdf
         "SpousesCU Partners SSN if filing jointly": get_address, # address text field
         "CountyMunicipality Code See Table page 50": @xml_document.at("ReturnHeaderState Filer USAddress CityNm")&.text,  # city / town text field
         "State": @xml_document.at("ReturnHeaderState Filer USAddress StateAbbreviationCd")&.text,
-        "ZIP Code": @xml_document.at("ReturnHeaderState Filer USAddress ZIPCd")&.text
+        "ZIP Code": @xml_document.at("ReturnHeaderState Filer USAddress ZIPCd")&.text,
+
+        # line 6 exemptions
+        "Check Box39": @xml_document.at("Exemptions SpouseCuRegular")&.text == "X" ? "Yes" : "Off",
+        "Check Box40": "Off",
+        "Domestic": get_line_6_exemption_count,
+        "x  1000": get_line_6_exemption_count * 1000,
```

Comment:
[pebble/dust] For this and the `x 1000_2` below, can we get this value from the XML instead of redoing the calculation here?

Reply:
This value is not stored in the XML, only the calculator. Guidance from CfA was to recalculate this value here since they don't want us pulling PDF values from the calculator.